cornflow/tests/unit/test_cases.py

```diff
@@ -272,3 +272,76 @@ def test_get_nonexistent_instance(self):
         self.get_one_row(
             self.url + "500" + "/", {}, expected_status=404, check_payload=False
         )
+
+    def test_update_one_instance(self):
```

Comment:
Copying and pasting code looks bad: specially if the names of the methods are not even modified (instance?)

If we're re-using code from the instances, I'd create a meta class for testing and use it for both. For ways of reusing tests for two classes, check `BaseTestCases.ListFilters` in the `custom_test_case.py` file. We use those tests in cases, instances and executions.